### PR TITLE
docker: specify --bind-mount-prefix=/host

### PR DESCRIPTION
**Pull request by giuseppe**
so that we are able to access all the paths on the host from the system container.

Depends on:

https://github.com/projectatomic/docker/pull/308

Let's not merge this yet, and wait for the change to be propagated into the packages first

### CODE REVIEW

File: docker-centos/config.json.template [hunk 1]
```diff
@@ -343,6 +343,17 @@
 		"mode=755"
 	    ]
 	},
+        {
+            "destination": "/host",
+            "options": [
+                "rbind",
+                "rshared",
+                "rw",
+                "mode=777"
```
Comment (rhatdan) [inline docker-centos/config.json.template]: As I stated in other PR, this concerns me.  What does this mode do?

**Comment by runcom** [comment]
:+1:

**Comment by rhatdan** [comment]
:+1:

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**